### PR TITLE
Awesome Books recreation with ES6 Modules

### PR DESCRIPTION
**Pull request by Rachelwebdev**
### In this project, I;

- Created ES6 modules, and divided my code into them
- Refactored normal functions to ES6 arrow functions
- Used Luxon library to set the current date and time
- Set up HTML,CSS and JavaScript linters
- Fixed eslint errors

### CODE REVIEW

File: modules/Book.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    const { id } = this;
+    books = books.filter((book) => {
+      if (book.id !== id) {
+        return true;
+      }
+      return false;
+    });
+    localStorage.setItem("books", JSON.stringify(books));
+  }
+}
```
Comment (mckent05) [inline modules/Book.js]: - I have this error when I try to remove a book from the list, Please fix this 🙏 
thdddh

File: modules/Book.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
```
Comment (mckent05) [inline modules/Book.js]: - Please remove all disabled linter from your project, remember linters help to ensure that we write code that is clean and easy to maintain. Please fix this 😄

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Rachelwebdev, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @Rachelwebdev, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔
## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/Book.js (hunk 1 above)
Comment (Rachelwebdev) [reply modules/Book.js]: Thanks for noticing this bug. I would fix it.

File: modules/Book.js (hunk 2 above)
Comment (Rachelwebdev) [reply modules/Book.js]: Okay, I would delete codes that disable linters. Thanks for the review

**Comment by Rachelwebdev** [comment]
Thanks for the Approval 👍

**Comment by Rachelwebdev** [comment]
Thanks for the Approval 👍

**Pull request merged**